src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
+import getToken from '../lib/tokens';
+import { Redirect } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 
 const Home = () => {
-  return <h1>Home</h1>;
+  // const [hasToken, setHasToken] = useState(false);
+
+  let history = useHistory();
+
+  const Token = function generateToken() {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    history.push('/list-view');
+
+    console.log(token);
+  };
+
+  // function fetchToken() {
+  //   localStorage.getItem('token');
+  //   history.push('/list-view');
+  // }
+
+  const existingToken = localStorage.getItem('token');
+
+  return (
+    <div>
+      <button onClick={Token}>Create a new list</button>
+      {existingToken ? <Redirect from="/" to="/list-view" /> : ''}
+      {
+        {
+          /* <input> below will be used at a later date  */
+        }
+        /* <input
+        placeholder="three work token"
+        for="inputToken"
+        onChange={fetchToken}
+      ></input> */
```

Comment:
On another note, I think it's best to not commit commented out blocks of code if we don't need them in this PR. I recommend keeping this code in your notes for future use, then when you're ready to use it, you can paste it in and commit it. Otherwise, the commit has a bunch of noise reviewers have to filter out before finding what we're actually reviewing.

Obviously there are always exceptions, so just use your best judgement. Think: _Does this have a chance to confuse someone reviewing my code?_ If so, it's best to leave it out, and include it when you really need it. 😄 

Otherwise, I love that you're thinking ahead! The further you can think ahead the better decisions you can make now.

Reply:
Commented out code removed! (It's tempting to use the code files as a notebook sometimes 😊)